tests/test_android.py

```diff
@@ -0,0 +1,63 @@
+import platformdirs
+
+if hasattr(platformdirs, "_android_folder"):
+
+    def test_backward_compatibility_basic():
```

Comment:
Use pytest skip marker instead, not this module-level if.  Better yet use mocking to ensure this exists.